lang/en.yml

```diff
@@ -5,7 +5,7 @@ en:
     DISPLAYED: Displayed
     NOT_DISPLAYED: 'Not displayed'
     ShowTitleLabel: Displayed
-    TitleLabel: 'Title (displayed if checked)'
+    TitleLabel: 'Title'
```

Comment:
Non-English languages will still have more context than this does. When we change English strings we usually must use a new translation key so it doesn't pollute non-English language packs. Can you please rename this? Something like `Title: 'Title'` would do